### ISSUE TITLE
数据集下载

### ISSUE THREAD

**Issue by liangjinggui** [xwgeng/WMT17-scripts#1]
请问该数据集有下载链接么？

**Comment by xwgeng** [comment]
Please refer to the website http://www.statmt.org/wmt17/translation-task.html

**Comment by liangjinggui** [comment]
> Please refer to the website http://www.statmt.org/wmt17/translation-task.html

你好~在prepare-wmt17en2zh.sh文件中，我看到关于数据处理部分，有使用到$tmp/$trainset.raw.$src，和$tmp/$trainset.raw.$tgt， 请问训练数据中的src文件和tgt文件在哪里呢？没看到脚本中有下载相关数据的连接

**Comment by xwgeng** [comment]
The training dataset is constructed by concatenating `News Commentary v12`, `UN Parallel Corpus v1.0` and `CWMT`, which can be downloaded using the link in aforementioned website.

**Comment by kkeleve** [comment]
Hello, I used this script to train on the 100w data set(casia2015 zh-en), and only got 21 BLEU,it is too far from 30+BLEU
I splite casia2015 data to train、test、valid as 95 : 2.5 : 2.5
I want to know how to adjust the parameters to achieve a better BLEU,and this is train script and train_log
[train_tc4.log](https://github.com/xwgeng/WMT17-scripts/files/7772501/train_tc4.log)

CUDA_VISIBLE_DEVICES=0 nohup fairseq-train ${data_dir}/data-bin \
  -a transformer --optimizer adam --source-lang ${src} --target-lang ${tgt} \
  --label-smoothing 0.1 --dropout 0.3 --max-tokens 4000 \
  --lr-scheduler inverse_sqrt --weight-decay 0.0001 \
  --criterion label_smoothed_cross_entropy --max-update 200000 \
  --warmup-updates 10000 --warmup-init-lr '1e-7' --lr '0.001' \
  --adam-betas '(0.9, 0.98)' --adam-eps '1e-09' --clip-norm 25.0 \
  --update-freq 4 --max-epoch 25 \
  --tensorboard-logdir ~/nmt/log/tensorboardlog_tc4 \
  --keep-last-epochs 2 --save-dir ${model_dir}/checkpoints_tc4 > ~/nmt/log/train_tc4.log 2>&1 &

**Comment by xwgeng** [comment]
The training dataset contains 20M sentence pairs (>> 100w), including `News Commentary v12`, `UN Parallel Corpus v1.0` and `CWMT` (including casia2015 zh-en). Besides, we utilizes newsdev2017 and newstest2017 as development and test sets respectively.
